specification/columns/skupricedetails.md

```diff
@@ -0,0 +1,52 @@
+# SKU Price Details
+
+The **SKU Price Details** column represents a list of relevant properties shared by all resources having the same [**SKU Price ID**](#skupriceid). These properties enable a practitioner to more easily calculate metrics such as total units of a service when it is not directly billed in those units (e.g. cores) and thus enables FinOps capabilities such as unit economics. They also help a practitioner understand the specifics of a **SKU Price ID**.
+
+
+The _SkuPriceDetails_ column adheres to the following requirements:
+
+* The _SkuPriceDetails_ column MUST be in [_KeyValueFormat_](#key-valueformat).
+* The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
+* The properties (both key and value) contained in the _SkuPriceDetails_ column MUST be shared across all resources having the same _SkuPriceId_.
+* The key for a property SHOULD remain consistent across comparable SKUs having that property and the values for this key SHOULD remain in a consistent format.
+* The _SkuPriceDetails_ column SHOULD NOT contain properties which are not applicable to the corresponding _SkuPriceId_.
+* The _SkuPriceDetails_ column MAY contain properties which are already captured in other dedicated columns.
+* Where _SkuPriceDetails_ contains a property with a value which is a numerical, the value provided MUST represent the value for a [_ConsumedQuantity_](#consumedquantity) of 1.
+* Properties included in _SkuPriceDetails_ SHOULD be immutable once included for a _SkuPriceId_.
```

Comment:
Can we make it clear that some properties are allowed to added - but we don't want existing properties to be changed.

Reply:
Modified in most recent commit